.github/workflows/test.yml

```diff
@@ -99,8 +99,8 @@ jobs:
     name: Test PiOS ${{ matrix.docker-tag }}
     runs-on: ubuntu-latest
     strategy:
-      matrix:
-        docker-tag: ['stretch-2018-03-13', 'buster-2021-05-28', 'buster-legacy-2022-04-07']
+      matrix:  # bullseye uses Python 3.9 which is not yet supported.
+        docker-tag: ['buster-2021-05-28', 'buster-legacy-2022-04-07']  # , 'bullseye-2022-01-28']
```

Comment:
Thanks @cclauss!

Could you reverse this change? I think I will fix the Stretch apt repositories URL in a different PR so that we can continue using it until we drop Python 3.5.

Reply:
The commit message says:
> Drop stretch because http://mirrordirector.raspbian.org/raspbian/dists/stretch/ is now empty.